src/Mobs/Monster.cpp

```diff
@@ -696,7 +696,7 @@ void cMonster::KilledBy(TakeDamageInfo & a_TDI)
 			break;
 		}
 	}
-	if ((a_TDI.Attacker != nullptr) && (!IsBaby()))
+	if (a_TDI.Attacker != nullptr)
```

Comment:
Does this patch cause baby animals to spawn experience orbs? If so is this expected behaviour?